### ISSUE TITLE
Full scrollback log not available

### ISSUE THREAD

**Issue by TheSonAlsoRises** [PonyChat/Shuo#69]
This thing:

![lawal4b](https://cloud.githubusercontent.com/assets/8591366/10013112/a4d32886-610e-11e5-9450-5179e4fed1e5.png)

I am unsure if this by design or a bug, but the scrollback only seems to feature a limited number of lines, making it impossible to read the entire log on fast-moving channels.

It appears I can only click on it once after which no additional lines will be loaded, forcing me to download the actual log file to read the conversations I missed.

**Comment by Xe** [comment]
Interesting. How many people are in that channel and how many lines do they send per hour (on average)?

**Comment by floogulinc** [comment]
Yeah, that's how it seems to work, It only lets you go back as far as what it keeps in RAM, which seems to be an extra 2 screens of messages (100 messages on the screen normally, 200 in the backlog, you end up with 300 total).

**Comment by TheSonAlsoRises** [comment]
@Xe: the numbers listed by floogulinc seem to check out, I have about 100 lines or so on the channels no matter the number of users.

@floogulinc: is there a way to change this behavior? I was (and still am) able to go back as far as I want with Shout, which is very practical.

**Comment by floogulinc** [comment]
@TheSonAlsoRises Oh you are? AFAIK we haven't changed anything related to that from Shout.

**Comment by TheSonAlsoRises** [comment]
I can test it again on Shout (and maybe you can too), but as far as I know there is no limit.

**Comment by Xe** [comment]
There was in [2c00acc7](https://github.com/PonyChat/Shuo/commit/2c00acc71dbc357177ae8235b64c4d785f84ddc0). Shuo was having OOM problems without this when a larger instance was deployed.

**Comment by floogulinc** [comment]
Oh, so if logging is turned on it will load more from the log file?

**Comment by Xe** [comment]
It should, [magic number here](https://github.com/PonyChat/Shuo/commit/2c00acc71dbc357177ae8235b64c4d785f84ddc0#diff-cf27c1d543e886c89cd9ac8b8aeaf05bR257)
